rpcs3/rpcs3qt/localized.h

```diff
@@ -22,13 +22,13 @@ class Localized : public QObject
 	const struct category // (see PARAM.SFO in psdevwiki.com) TODO: Disc Categories
 	{
 		// PS3 bootable
-		const QString app_music = tr("App Music");
-		const QString app_photo = tr("App Photo");
-		const QString app_tv    = tr("App TV");
-		const QString app_video = tr("App Video");
+		const QString app_music = tr("Music App");
+		const QString app_photo = tr("Photo App");
+		const QString app_tv    = tr("TV App");
+		const QString app_video = tr("Video App");
 		const QString bc_video  = tr("Broadcast Video");
 		const QString disc_game = tr("Disc Game");
-		const QString hdd_game  = tr("HDD Game");
+		const QString hdd_game  = tr("Downloaded Game");
```

Comment:
I don't really like this change. We've called this category HDD game forever, and downloaded game sounds like piracy or whatever.

Reply:
Feel free to change :)